flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/codegen/calls/ScalarOperatorGens.scala

```diff
@@ -935,6 +936,12 @@ object ScalarOperatorGens {
       operand: GeneratedExpression,
       targetType: LogicalType)
     : GeneratedExpression = {
+
+    ctx.addReusableHeaderComment("Configuration")
```

Comment:
let's drop those two lines. every added header comment should be self contained otherwise you will end up with comments like:
```
Configuration
--------------
config option 1
Other comment
----------------
other comment
config option 2

```